### DISCUSSION TITLE
Images Location

### DISCUSSION THREAD

**Discussion by Timelessprod** [t1m0thyj/WinDynamicDesktop#297] [Q&A] [answered]
Hi,

I wonder if someone can tell me where are the images used in a theme on WinDynamicDesktop ?

I'm using the Microsoft Store version on Windows 10.

Thanks

**Comment by t1m0thyj** [comment, marked as answer]
They are in the "themes" folder inside the location where WDD is installed: https://github.com/t1m0thyj/WinDynamicDesktop/wiki/Troubleshooting#finding-where-app-is-installed
